core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -524,6 +529,15 @@ class SparkContext(config: SparkConf) extends Logging {
     executorEnvs ++= _conf.getExecutorEnv
     executorEnvs("SPARK_USER") = sparkUser
 
+    _shuffleDriverComponents = ShuffleDataIOUtils.loadShuffleDataIO(config).driver()
+    _shuffleDriverComponents.initializeApplication().asScala.foreach { case (k, v) =>
+      val key = ShuffleDataIOUtils.SHUFFLE_SPARK_CONF_PREFIX + k
+      if (_conf.contains(key)) {
+        logDebug(s"Setting shuffle spark config ${key} to ${v}")
```

Comment:
sorry when I first proposed logging this, I had thought this was getting logged after the environment update event is created.   Now I think that should be enough.  Also @mccheah has a good point, if we are logging we have to make sure we go through redaction.

Reply:
I logged only the key, perhaps that's enough? I'm not sure what redaction is

Reply:
See https://spark.apache.org/docs/latest/configuration.html#runtime-environment - specifically `spark.redaction.regex`, see also `Utils#redact`.